make/jdk/src/classes/build/tools/tzdb/TzdbZoneRulesProvider.java

```diff
@@ -304,7 +305,7 @@ private void parse(String[] tokens, int off) {
             month = parseMonth(tokens[off++]);
             if (off < tokens.length) {
                 String dayRule = tokens[off++];
-                if (dayRule.startsWith("last")) {
+                if (dayRule.regionMatches(true, 0, "last", 0, 4)) {
```

Comment:
This isn't correct, as per the mailing list:

> > can "last" contain uppercase letters, or
> > does it have to be exactly "last"?
> 
> In that case the name is "lastSunday", and it can be abbreviated
> "lastSu" or "Lastsu" or "LASTSUNDA" or whatever.

Reply:
Why is it not? IIUC, abbreviation should be unambiguous, so "last" part is always case-insensitive 4 character length.

Reply:
Sorry, I assumed this regionMatches was the same as the others. "last" is always "last" and never abbreviated.